frontend/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -658,6 +700,44 @@ export function parseNml(
             }
             break;
           }
+          case "userBoundingBox": {
+            const userBoundingBoxId = _parseInt(attr, "id");
+            const boundingBoxObject = {
+              topLeft: [
+                _parseInt(attr, "topLeftX"),
+                _parseInt(attr, "topLeftY"),
+                _parseInt(attr, "topLeftZ"),
+              ],
+              width: _parseInt(attr, "width"),
+              height: _parseInt(attr, "height"),
+              depth: _parseInt(attr, "depth"),
+            };
+            if (
+              boundingBoxObject.topLeft[0] == null ||
+              boundingBoxObject.topLeft[1] == null ||
+              boundingBoxObject.topLeft[2] == null ||
+              boundingBoxObject.width == null ||
+              boundingBoxObject.height == null ||
+              boundingBoxObject.depth == null
+            ) {
+              throw new NmlParseError(
+                `${messages["nml.incomplete_bounds"]} User bounding box id: ${userBoundingBoxId}.`,
+              );
+            }
```

Comment:
As you are not specifying a default value, `_parseInt` will take care of this null check and throw an error if one of the values is not supplied. You should be able to remove these lines.